src/web/lib/components/PaginatedThemeSelector/index.scss

```diff
@@ -0,0 +1,104 @@
+@import '../../common.scss';
+
+.theme-selector {
+  h2 {
+    grid-column: 1 / -1;
+    margin: 0;
+  }
+
+  footer.page-selector {
+    display: flex;
+    grid-column: 1 / -1;
+    justify-content: center;
+    margin: 0;
+
+    button {
+      color: #000;
+      background-color: transparent;
+      border: 0;
+      cursor: pointer;
+      margin: 2px;
+      height: 18px;
+      overflow: hidden;
+      text-indent: 10px;
+      white-space: nowrap;
+      width: 10px;
+    }
+    button.page {
+      text-indent: 0;
+      &::before {
+        content: "\25cf";
```

Comment:
Behold my terrible unicode CSS. Didn't see SVGs for the circle page indicators [in assets](https://github.com/mozilla/testpilot-assets/tree/master/ThemesRFun/00_Assets/preset-themes), so I threw this in there for now.